clients/spark/core/src/main/scala/io/treeverse/clients/GarbageCollector.scala

```diff
@@ -301,17 +301,17 @@ object GarbageCollector {
   }
 
   def main(args: Array[String]) {
-    if (args.length != 2) {
```

Comment:
I would change this to a check by storage type instead, and exit with an error in case the arguments are invalid

Reply:
In order to get the storage type (for now) we need to initiate lakeFS client.. Do you think we should do it and fail the job?

Reply:
Changing it since there is no use of the arguments before initiating the client.